static/app/components/replays/breadcrumbs/replayTimeline.tsx

```diff
@@ -42,9 +43,26 @@ function ReplayTimeline({}: Props) {
   const chapterFrames = replay.getChapterFrames();
   const networkFrames = replay.getNetworkFrames();
 
+  // timeline size is 300% larger
+  const timelineWidthPercentage = 300;
+
+  // start of the timeline is in the middle
+  const timelineTranslatePercentage = 50 / timelineWidthPercentage;
+
+  const translatePercentage =
+    (timelineTranslatePercentage -
+      (currentTime > durationMs ? 1 : divide(currentTime, durationMs))) *
+    100;
+
   return hasNewTimeline ? (
-    <PanelNoMarginBorder ref={panelRef} {...mouseTrackingProps}>
-      <Stacked ref={stackedRef}>
+    <VisiblePanel ref={panelRef} {...mouseTrackingProps}>
+      <Stacked
+        style={{
+          width: `${timelineWidthPercentage}%`,
+          transform: `translate(${translatePercentage}%, 0%)`,
```

Comment:
There's a `toPercent` helper that might be good to use here: static/app/utils/number/toPercent.tsx

not required, and it might truncate the numbers too much which makes the scrolling more choppy.